src/main/java/org/quiltmc/loader/impl/metadata/qmj/V1ModMetadataReader.java

```diff
@@ -237,6 +238,21 @@ private static V1ModMetadataImpl readFields(JsonLoaderValue.ObjectImpl root) {
 				}
 			}
 
+			@Nullable
+			JsonLoaderValue intermediateMappingsValue = quiltLoader.get("intermediate_mappings");
+
+			if (intermediateMappingsValue != null) {
```

Comment:
If it is null, we can assume that the mappings are hashed (because that is the default).

Reply:
I'm assuming you mean intermediary instead of hashed here as hashed mappings are not yet supported. I am just going to add an else block to fallback to intermediary mappings so that parsing exceptions still get thrown if a mod contains an unsupported value for `intermediate_mappings`.